.github/workflows/main.yml

```diff
@@ -329,6 +330,12 @@ jobs:
           optimize_flags='-O3 -ffast-math -fno-finite-math-only'
       - name: Test Cantera
         run: scons test show_long_tests=yes verbose_tests=yes
+      - name: Test Install
+        run: |
+          scons install
+          python -c 'import cantera as ct; print(ct.__version__)'
+          ck2yaml --input=test/data/h2o2.inp --output=h2o2-test.yaml
+          cti2yaml test/data/air-no-reactions.cti air-no-reaction-test.yaml
```

Comment:
I think the actual tests of the functionality should be in a separate step. I also think some good tests would be that files end up where they're supposed to, for example, `ct.__file__ == $CONDA_PREFIX/.../cantera` and `test -f $CONDA_PREFIX/lib/libcantera.so`